### PR TITLE
fixed links and description

### LINKED ISSUE #11: Link redirects
Checked the links on the page and several are being redirected so they need to be updated on the site:

http://www.outreachy.org/ -> https://www.outreachy.org/
http://up-for-grabs.net/ ->  https://up-for-grabs.net/
https://www.meetup.com/opensourceladies/  ->  https://www.meetup.com/opensourcediversity/

### PR DESCRIPTION
**Pull request by 22nds**
Added https on two links and changed the meetup link + updated description for Open Source Diversity. 

Closes #11 

@jancborchardt please review :)

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -178,10 +178,10 @@ <h3>Common Voice</h3>
 			<h2>Spaces</h2>
 			<p>Find a local space or meetup where you can work with like-minded people!</p>
 			<div class="row">
-				<a class="col-md-4" href="https://www.meetup.com/opensourceladies/" target="_blank">
+				<a class="col-md-4" href="https://www.meetup.com/opensourcediversity/" target="_blank">
 					<img alt="" src="img/projects/opensourceladies.jpg" />
-					<h3>Open Source Ladies</h3>
-					<p>Monthly meetup in Berlin for women* who want to contribute to open source projects.</p>
+					<h3>Open Source Diversity</h3>
```
Comment (jonatoni) [inline index.html]: Based on the conversation we had at #1 Open Source Ladies will be renamed to Open Source Diversity Berlin.

**Comment by jonatoni** [comment]
I had only one comment, when you fix it we can merge this PR. Thanks @22nds :)

**Comment by 22nds** [comment]
@jonatoni Berlin added :)

**Comment by jonatoni** [comment]
@22nds I think you added Berlin at the header? I meant to change Open Source Ladies to Open Source Diversity Berlin. Sorry for the confusion.

**Comment by jancborchardt** [comment]
Good stuff! :) Looks good except @jonatoni’s comment – indeed the »Berlin« should only be added to the entry which was »Open Source Ladies« before.

Also there’s some conflicts with the master branch, probably because I added `rel="noopener"` to all the links. Sorry about that, it’s probably easy to fix with a `git pull --rebase` and then fixing the conflicts, but let me know if you need help.

(And btw, since you are part of the organization, feel free to use branches directly in this repository, you don’t need your own fork. :)

**Comment by 22nds** [comment]
@jancborchardt @jonatoni All changes made so you can gloriously push the button and merge it ;) Thanks for the instructions!

File: index.html [hunk 2]
```diff
@@ -2,7 +2,7 @@
 <html lang="en">
 	<head prefix="og: http://ogp.me/ns#">
 		<meta charset="utf-8">
-		<title>Open Source Diversity</title>
+		<title>Open Source Diversity</title
```
Comment (jancborchardt) [inline index.html]: The ending closing `>` is missing. :)

**Comment by jonatoni** [comment]
Great job @22nds :)

**Comment by 22nds** [comment]
@jonatoni & @jancborchardt thank you for your patience ;)

**Comment by jancborchardt** [comment]
@22nds no worries, and welcome to the team!

**Pull request merged**